lib/helpers/protobom.js

```diff
@@ -23,19 +31,23 @@ const stringifyIfNeeded = (bomJson) => {
  */
 export const writeBinary = (bomJson, binFile) => {
   if (bomJson && binFile) {
-    let bomObject;
+    let bomSchema;
     if (+bomJson.specVersion === 1.6) {
-      bomObject = new cdx_16.Bom();
+      bomSchema = cdx_16.BomSchema;
     } else {
-      bomObject = new cdx_15.Bom();
+      bomSchema = cdx_15.BomSchema;
     }
     writeFileSync(
       binFile,
-      bomObject
-        .fromJsonString(stringifyIfNeeded(bomJson), {
+      toBinary(
+        bomSchema,
+        fromJsonString(bomSchema, stringifyIfNeeded(bomJson), {
```

Comment:
I am not particularly proud of this snippet, but things do not always directly translate between jsonschema and protobuf correctly. On occasions, there could be bugs in the spec.